InternetShop/src/main/java/dao/UserDao.java

```diff
@@ -2,11 +2,15 @@
 
 import model.User;
 
+import java.util.Map;
+
 /**
  * Created by Igor on 10/8/2019.
  */
 public interface UserDao {
-    void setUserIntoDatabase(User user);
-    User getUserFromDatabase();
-    void checkUserInDatabase(User user);
+    void setUserIntoDatabase(User user); //create
+    User getUserByName(String name); //read
+    Map<String, User> getAll();
+    void checkUserInDatabase(User user); //update
```

Comment:
In my opinion, this method should just check if there is such a user in the database, and return true or false
boolean isUserInDatabase(User user);